types/schemas/preApplication/data/Applicant.ts

```diff
@@ -0,0 +1,36 @@
+import {Email} from '../../../shared/utils';
+import {
+  Agent,
+  Ownership,
+  SiteContact,
+  UserAddress,
+} from '../../application/data/Applicant';
```

Comment:
In the future, these should be refactored & imported from `../shared` types rather than the application schema - but it'll be easier to do that refactor once example payloads & tests are in place for each schema type - so stay tuned for a future PR! (Also helps reduce noise here, as the refactor will touch many files)